src/main/java/no/nav/familie/integrasjoner/tilgangskontroll/FaviconController.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.familie.integrasjoner.tilgangskontroll
+
+import no.nav.security.token.support.core.api.Unprotected
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class FaviconController {
+    @GetMapping("favicon.ico")
+    @Unprotected
+    fun dummyFavicon() {
```

Comment:
Dummy favicon.ico for å forhindre No static resource favicon.ico feil i loggene ved f.eks. oppslag in internal/metrics med en browser, siden spring ikke lenger leverer med en default favicon